.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,151 +2,254 @@
-1.  - [ ] [Internal] Push the servicing branch to your GitHub fork and submit a source-build PR.
-1.  - [ ] [Internal] If a build hits `fatal: reference is not a tree: cb5f173b`, [make the commit available on a fork and switch to it temporarily](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/make-commit-available.md).
-1.  - [ ] [Internal] Add source-build team as reviewers.
-1.  - [ ] [Internal] When CI is green and two reviewers approve, merge.
-      - Avoid squash/rebase: nice to preserve commit hashes. However, there are no known dependencies on *source-build* commits being preserved.
-1.  - [ ] Create and push tags on the post-merge commit. [/Documentation/servicing/tagging.md](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/tagging.md)
-1.  - [ ] Create a GitHub release on the tag with SDK versioning.
+        - Do not remove from subrepos. The build infra removes these sources
+          automatically while modifying each subrepo nuget.config file.
+1. - [ ] Push the servicing branch to your GitHub fork and submit a
+         source-build PR.
+1. - [ ] If a build hits `fatal: reference is not a tree: cb5f173b`,
+        [make the commit available on a fork and switch to it temporarily](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/make-commit-available.md).
+1. - [ ] Add crummel, MichaelSimons, and lbussell as reviewers.
+1. - [ ] When CI is green and two reviewers approve, merge.
+      - Avoid squash/rebase: nice to preserve commit hashes. However, there
+        are no known dependencies on *source-build* commits being preserved.
+1. - [ ] Create and push tags on the post-merge commit. [/Documentation/servicing/tagging.md](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/tagging.md)
+1. - [ ] Create a GitHub release on the tag with SDK versioning.
       - Go to <https://github.com/dotnet/source-build/tags>
       - Expand the SDK-versioned tag's message and copy it
-      - Click <kbd>...</kbd> on the right on the SDK-versioned tag and press "Create release"
+      - Click `...` on the right on the SDK-versioned tag and press "Create release"
       - Paste into the release title
       - Click submit.
-1.  - [ ] Notify distro maintainers and partners about the release tags
-1.  - [ ] [3.1] Download merged Private.SourceBuilt.Artifacts.XX.tar.gz files from the prepare-artifacts leg in source-build-rolling-CI.
-1.  - [ ] [3.1] Upload merged Private.SourceBuilt.Artifacts.XX.tar.gz file to dotnetcli account source-built-artifacts blob container.
-      - Never overwrite a tarball. Use a sub-patch version if one already exists, e.g. `0.1.0-3.1.105.1`.
-1.  - [ ] If there's a respin (for any reason: source-build infra fixes or upstream repo problems):
+1. - [ ] [Source-build team] Notify distro maintainers and partners about
```

Comment:
What channels are used for this?  Is there a template for the notification?

Reply:
Added Slack and GitHub and the recommended information.